packages/vertica-nodejs/package.json

```diff
@@ -49,6 +49,6 @@
   ],
   "license": "Apache-2.0",
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 14.0.0"
```

Comment:
Why was this changed here?   I checked the spec and we did decided to support node 14 and higher.   I'm okay with the change, but want to double check to make absolutely certain this change won't cause problems.

Reply:
The 'engines' specification in package.json is just a place to specify the version of node that our stuff works on. It has no impact unless you also have the engineStrict flag set to true. As it is with the change I've just made installing with npm if your node version is < 14 will give the user a warning, but will not keep them from installing. If the engine strict flag was set to true it would give them an error and say it's not compatible

This is just another way to say we think you should use node 14+, but we won't force you to